bubblesort.rb

```diff
@@ -0,0 +1,15 @@
+def bubble_sort_by(array)
+  laps = 0
+
+  while laps < array.size - 1
+
+    (0...array.size - 1).each do |item|
+      array[item], array[item + 1] = array[item + 1], array[item] if array[item] > array[item + 1]
+    end
+
+    laps += 1
+  end
+  array
+end
```

Comment:
- [ ] bubble_sort_by method is implemented and working (Screenshot from Odin)
- [ ] sorts an array of numbers in increasing order
```ruby
unsorted = (1..10).to_a.reverse!
bubble_sort(unsorted) == unsorted.sort # should return true
```
- [ ] sorts an array by accepting a block
```ruby
unsorted = %w[hi hello hey]
bubble_sort_by(unsorted) { |left, right| left.length - right.length } == %w[hi hey hello] # should return true
```
- [ ] raises an error if block is not given
```ruby
unsorted = %w[hi hello hey]
bubble_sort_by(unsorted) == raise_error # should return true
```